include/ginkgo/core/base/exception_helpers.hpp

```diff
@@ -147,6 +148,22 @@ inline dim<2> get_size(const T& op)
 inline dim<2> get_size(const dim<2>& size) { return size; }
 
 
+template <typename T>
+inline batch_dim<2> get_batch_size(const T& op)
+{
+    return op->get_size();
```

Comment:
```suggestion
    return op->get_size();
```
should it still use `get_batch_size()`?

Reply:
No, the BatchLinOp getter is also `get_size()` and it returns a `batch_dim` object.